app/assets/javascripts/dashboard/views/dash/logged_time_view.js

```diff
@@ -0,0 +1,130 @@
+// @flow
+
+import React from "react";
+import moment from "moment";
+import FormatUtils from "libs/format_utils";
+import Request from "libs/request";
+import c3 from "c3";
+
+function LoggedTimeList({ items }) {
+  console.log(items);
+  return (
+    <table className="table-striped table-hover table">
+      <thead>
+        <tr>
+          <th>Month</th>
+          <th>Worked Hours</th>
+        </tr>
+      </thead>
+      <tbody>
+        {items.map((item, index) =>
+          <tr key={index}>
+            <td>
+              {moment(item.interval).format("MM/YYYY")}
+            </td>
+            <td>
+              {FormatUtils.formatSeconds(item.time.asSeconds())}
+            </td>
+          </tr>,
+        )}
+      </tbody>
+    </table>
+  );
+}
+
+export default class LoggedTimeView extends React.PureComponent {
+  props: {
+    userID: ?string,
+  };
+
+  state: {
+    timeEntries: Array<Object>,
+  } = {
+    timeEntries: [],
+  };
+
+  componentDidMount() {
+    this.fetch();
+  }
+
+  componentDidUpdate() {
+    this.renderGraph();
+  }
+
+  async fetch(): Promise<void> {
+    const url = this.props.userID
+      ? `/api/users/${this.props.userID}/loggedTime`
+      : "/api/user/loggedTime";
+
+    Request.receiveJSON(url).then(response => {
+      const timeEntries = response.loggedTime
+        .map(entry => {
+          const interval = entry.paymentInterval;
+          return {
+            interval: moment(`${interval.year} ${interval.month}`, "YYYY MM"),
+            time: moment.duration(entry.durationInSeconds, "seconds"),
+            months: interval.year * 12 + interval.month,
+          };
+        })
+        .sort((a, b) => b.months - a.months);
+
+      this.setState({
+        timeEntries,
+      });
+    });
+  }
+
+  renderGraph = () => {
+    // Only render the chart if we have any data.
+    const timeEntries = this.state.timeEntries;
+    if (timeEntries.length > 0) {
+      const dates = timeEntries.map(item => item.interval.toDate());
+      const monthlyHours = timeEntries.map(item => item.time.asHours());
+
+      c3.generate({
```

Comment:
I wonder if there is a react wrapper for c3? If not, we could always use another lightweight react charting lib. Thoughts?